src/main/java/com/st/lms/controller/BorrowerController.java

```diff
@@ -0,0 +1,88 @@
+package com.st.lms.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.st.lms.service.BorrowerService;
+import com.st.lms.models.*;
+import com.st.lms.dto.*;
+import com.st.lms.exception.*;
+
+@RestController
+public class BorrowerController {
+
+	@Autowired
+	BorrowerService borrowerService;
+	
+	//return list of branches where the borrower has borrowed books from
+	@GetMapping("/cardNo/{cardNo}/libraries")

```

Comment:
cardNo is not really a resource. It is a identification of a resource (borrower), I would recommend changing this.
```suggestion
	@GetMapping("/borrowers/{cardNo}/libraries")
```

Reply:
You should also make this change in the rest of this file.

Reply:
:+1:, except that the resource should be singular (`borrower`, not `borrowers`).